util/containers/opencast-cors-proxy-nginx.conf

```diff
@@ -4,7 +4,7 @@ server {
 
     # Basic open CORS for all domains. Don't use this in production!
     add_header Access-Control-Allow-Origin $http_origin always;
-    add_header Access-Control-Allow-Methods 'GET, POST, OPTIONS' always;
+    add_header Access-Control-Allow-Methods 'GET, POST, PUT, OPTIONS' always;
```

Comment:
Uhwee I forgot to remove this... was using it for the OC api call but since that only works with `ROLE_ADMIN`, and we decided to rather implent this via some Tobira backend api, I think this can be removed again.

Reply:
But it also doesn't hurt. Might be useful at some point. It's just for local containers, so no harm done to any users of Tobira.